README.md

```diff
@@ -44,6 +45,36 @@ To run the tests
     make test
 
 
+.gitAttributes
+--------------
+
+Like in linguist you can override the strategies via `.gitattributes` file.
+Add a `.gitattributes` file to the directory and use the enry matchers `enry-documentation`,`enry-language` or `enry-vendored` to do the override.
```

Comment:
I don't know how `linguist` works, but why are we using specific enry matchers instead of reusing the linguist ones? Or are we adding these enry matchers _on top of_ the linguist ones?